scripts/arm32_ci_script.sh

```diff
@@ -203,9 +264,17 @@ do
             exit_with_error "--buildConfig can be only Debug or Release" true
         fi
         ;;
+    --arm)
+        __ARMRootfsImageBase="rootfs-u1404.ext4"
```

Comment:
Are we planning to use this script to mount the Arm emulator, given that the build is now being done in Docker image?